docs/examples/helm-module/README.md

```diff
@@ -0,0 +1,124 @@
+# Use Helm chart as capability module
+
+Here is an example of how to use Helm chart as workload capability module.
+
+## Install fluxcd/flux2 as dependencies
+
+This feature depends on several CRDs and controllers from [fluxcd/flux2](https://github.com/fluxcd/flux2), so we prepared a simplified Helm chart to install dependencies.
+
+It's worth noting that, flux2 doesn't offer an official Helm chart to install.
+And this chart only includes minimum dependencies this feature relies on, not all of flux2.
+
+```shell
+helm repo add flux2 https://captainroy-hy.github.io/helm-flux2/
+helm repo update
+helm install --create-namespace -n flux-system flux-system flux2/helm-flux2
+```
+
+## Write WorkloadDefinition 
+Here is an example `WorkloadDefinition` with only required data of a Helm module.
+
+Comparing to existing workload definition based on CUE template, several points worth attention in Helm module.
+
+- `.spec.definitionRef` is required to indicate the workload GVK in your Helm chart. For example, in our sample chart, the core workload is `deployments.apps/v1`.
+- `.spec.schematic.helm.template` is a string containing Helm release & repository definition in YAML format. 
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: webapp-chart
+  annotations:
+    definition.oam.dev/description: helm chart for webapp
+spec:
+  definitionRef:
+    name: deployments.apps
+    version: v1
+  schematic:
+    helm:
+      release:
+        chart:
+          spec:
+            chart: "podinfo"
+            version: "5.1.4"
+      repository:
+        url: "https://captainroy-hy.github.io/podinfo/"
+```
+Specifically, the definition follows the APIs from `fluxcd/flux2`, [HelmReleaseSpec](https://github.com/fluxcd/helm-controller/blob/main/docs/api/helmrelease.md) and [HelmRepositorySpec](https://github.com/fluxcd/source-controller/blob/main/docs/api/source.md#source.toolkit.fluxcd.io/v1beta1.HelmRepository).
+However, the fields shown in the sample are almost enough to describe a Helm chart release and its repository.
+
+## Define Application & Deploy
+
+Here is an example `Application`.
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: Application
+metadata:
+  name: myapp
+  namespace: default
+spec:
+  components:
+    - name: demo-podinfo 
+      type: webapp-chart # workload definiition
+      settings: # Helm chart values
+        replicaCount: 2
+```
+Diff from applications using CUE module workload, Helm module workload will use data in `settings` as the [Helm chart values](https://github.com/captainroy-hy/podinfo/blob/master/charts/podinfo/values.yaml).
```

Comment:
We should explain how to fill the settings instead of telling the difference with CUE. For example, tell users that they can read the README of chart and the arguments are totally align with values.yaml of the chart.

Reply:
Got it. I will refine the README.md as well as the demo example.